06-func.md

```diff
@@ -52,136 +52,16 @@ Let's try running our function.
 Calling our own function is no different from calling any other function:
 
 ~~~ {.python}
-print 'freezing point of water:', fahr_to_kelvin(32)
-print 'boiling point of water:', fahr_to_kelvin(212)
+print('freezing point of water:', fahr_to_kelvin(32))
+print('boiling point of water:', fahr_to_kelvin(212))
 ~~~
 ~~~ {.output}
 freezing point of water: 273.15
-boiling point of water: 273.15
+boiling point of water: 373.15
 ~~~
 
```

Comment:
Let's put a comment in saying that in earlier version of python this returns a truncated integer and see how the lesson does without the bug lesson here.